src/detector.ts

```diff
@@ -0,0 +1,7 @@
+import { PhishingDetector as MetaMaskPhishingDetector, PhishingDetectorResult } from "@metamask/phishing-controller";
+
+export default class PhishingDetector extends MetaMaskPhishingDetector {
+    public check(hostname: string): PhishingDetectorResult {
+        return super.check(`https://${hostname}`);
```

Comment:
@samczsun could you explain what is going on here?

My (limited) understanding is that you are ripping out `detector.js` in order to use the PhishingDetector from the Phishing Controller itself. And why do we need to extend `check` and add `https://` as a prefix here?

Reply:
The reason is that this codebase expects  `check` to take in a hostname (as per the tests, at least). However, the `PhishingDetector` from `@metamask/phishing-controller` expects `check` to take a URL. To maintain compatibility for users who are importing this package, we extend the class and rewrite the hostname into a url